Add quickstart to README with link to latest

dkristian opened the pull request:
Inspired by @toddmowen adding a link to the version in the userguide, I think it's also useful to link it from the readme.

toddmowen commented:
Agree that README is the first place I'd look for the link.

But why are the instructions different to what's in USERGUIDE?

Maybe we should put the sbt instructions in README, and just link to them from USERGUIDE.

dkristian commented:
@toddmowen Good point. Now that `scalding` depends on `tools` to write the metadata with the features to the sinks, it's sufficient to just depend on `scalding`. I've changed the USERGUIDE to refer to the README, which has been updated. I've also split out the plugin configuration to a new section in the USERGUIDE that explains how to use it to generate metadata. Ready for review again.

dkristian commented:
I've also piggy-backed a USERGUIDE correction onto this PR (relating to how aggregation is actually done now).

toddmowen commented on USERGUIDE.markdown:
```diff
@@ -1451,6 +1437,26 @@ With blank features, the usual metadata artifacts are generated even though
+[here](http://commbank.artifactoryonline.com/commbank/api/search/latestVersion?g=au.com.cba.omnia&a=coppersmith-core_2.11).
+
+Once configured, metadata can be generated by running the
+`metadata:export` task from SBT. The location of the generated file at
```
Missing words "will be logged" in this sentence.

toddmowen commented:
:+1: apart from typo

Good call moving the plugin to the Advanced section. It's not something that newbies need to try and contend with.
